client/src/components/Register.js

```diff
@@ -0,0 +1,130 @@
+            password
+        };
+
+        const init = {
+            method: 'POST', // GET by default
+            headers: {
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify(newUser)
+        };
+
+        fetch('http://localhost:5000/create_account', init)
+            .then(response => {
+                if (response.status === 201 || response.status === 400) {
+                    return response.json();
+                }
+                return Promise.reject('Something unexpected went wrong :)');
+            })
+            .then(data => {
+                if (data.id) {
+
+
+                    const init = {
+                        method: 'POST', // GET by default
+                        headers: {
+                            'Content-Type': 'application/json'
+                        },
+                        body: JSON.stringify(newUser)
+                    };
+
+                    fetch('http://localhost:5000/authenticate', init)
+                        .then(response => {
+                            if (response.status === 200) {
+                                return response.json();
+                            } else if (response.status === 403) {
+                                return null;
+                            }
+                            return Promise.reject('Something unexpected went wrong :)');
+                        })
+                        .then(data => {
+                            if (data) {
+                                auth.login(data.jwt_token);
+                                history.push('/home');
+                            } else {
+                                
+                                setErrors(['Login failure.']);
+                            }
+                        })
+                        .catch(error => console.log(error));
+                } else {
+                    
+                    setErrors(data.messages)
+                }
+            })
+            .catch(error => console.log(error));
+    };
+    return (
+        <main>
+            <h2>Login</h2>
+            <Errors errors={errors} />
+            <form onSubmit={formSubmitHandler}>
+                <div className="form-group">
+                    <label htmlFor="username">Username:</label>
+                    <input className="form-control" type="text" id="username" name="username"
+                        value={username} onChange={usernameOnChangeHandler} />
+                </div>
+                <div className="form-group">
+                    <label htmlFor="password">Password:</label>
+                    <input className="form-control" type="password" id="password" name="password"
+                        value={password} onChange={passwordOnChangeHandler} />
+                </div>
+                <div className="form-group">
+                    <label htmlFor="password">Confirm Password:</label>
```

Comment:
Getting an error in the console while running it about the non-unique id of  [id="password"] I would change this over to confirmPassword on the htmlfor, id, and name.